docs/admin/system-information.rst

```diff
@@ -2226,6 +2226,59 @@ the database.
 |              | for the specific database object                |             |
 +--------------+-------------------------------------------------+-------------+
 
+.. _sys-sessions:
+
+Sessions
+========
+
+.. list-table::
+    :header-rows: 1
+    :widths: auto
+    :align: left
+
+    * - Column Name
+      - Description
+      - Return Type
+    * - ``id``
+      - The unique identifier of the session within the ``handler`` node
+      - ``INTEGER``
+    * - ``auth_user``
+      - The user which was authenticated for the session
+      - ``TEXT``
+    * - ``session_user``
+      - The user, possibly different than the ``auth_user`` which is currently
+        active in the session, see: :ref:`ref-set-session-authorization`
+      - ``TEXT``
+    * - ``handler_node``
+      - The name of the node on which the session is created
+      - ``TEXT``
+    * - ``system_session``
+      - A flag, which when ``TRUE`` denotes that the session is opened
+        internally to execute SQL statements, e.g.: to gather table statistics,
+        see: :ref:`stats.service.interval`
```

Comment:
I'd be inclined to drop this column and hide system sessions from the output - this is to some degree leaking implementation details.

Reply:
Currently, the system sessions are used 1. to add implicitely some column, 2. to gather the stats.
The reason I added it is for 2. but this can been seen also in jobs/jobs_logs.
So, I agree, I will filter them out.